### PR TITLE
[16.0] [MIG] hr_attendance_autoclose

### PR DESCRIPTION
**Pull request by HekkiMelody**
Depends on https://github.com/OCA/hr-attendance/pull/112

### CODE REVIEW

**Comment by tafaRU** [comment]
@aleuffre thanks for your PR!
Looks good overall, but there are some commit generated by bots or weblate to be squashed (Ref. https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate).

**Review by FratiuG — APPROVED** [review]
functional test ok

**Comment by HekkiMelody** [comment]
> @aleuffre thanks for your PR! Looks good overall, but there are some commit generated by bots or weblate to be squashed (Ref. https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate).

Should be fine now

**Review by tafaRU — APPROVED** [review]

File: hr_attendance_autoclose/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#. Go to *Settings > Company > Attendances*.
```
Comment (tafaRU) [inline hr_attendance_autoclose/readme/USAGE.rst]: This is no longer true, now we have:
image

Could you please modify it?

**Review by tafaRU — CHANGES_REQUESTED** [review]

File: hr_attendance_autoclose/readme/USAGE.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#. Go to *Settings > Company > Attendances*.
+#. Go to *Attedances > Configuration > Configuration*.
```
Comment (tafaRU) [inline hr_attendance_autoclose/readme/USAGE.rst]: s/Attedances/Attendances

**Review by tafaRU — APPROVED** [review]

**Pull request merged**